scripts/processing/08-make_cov.py

```diff
@@ -31,7 +31,7 @@ def run_covariance(subject_id, tsss=False):
                             % (subject, l_freq))
     print('  Computing regularized covariance')
     epochs = mne.read_epochs(fname_epo, preload=True)
-    cv = ShuffleSplit(3, random_state=random_state)
+    cv = KFold(3, random_state=random_state)
```

Comment:
I think the docs in MNE are wrong:

http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html

Can you confirm and open a PR to fix them?